backend/src/main/java/com/festago/presentation/StaffController.java

```diff
@@ -0,0 +1,50 @@
+package com.festago.presentation;
+
+import com.festago.auth.annotation.Staff;
+import com.festago.auth.application.StaffAuthService;
+import com.festago.auth.dto.StaffLoginRequest;
+import com.festago.auth.dto.StaffLoginResponse;
+import com.festago.entry.application.EntryService;
+import com.festago.entry.dto.TicketValidationRequest;
+import com.festago.entry.dto.TicketValidationResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/staff")
+@Tag(name = "스태프 요청")
+public class StaffController {
+
+    private final StaffAuthService staffAuthService;
+    private final EntryService entryService;
+
+
+    public StaffController(StaffAuthService staffAuthService, EntryService entryService) {
+        this.staffAuthService = staffAuthService;
+        this.entryService = entryService;
+    }
+
+    @PostMapping("/login")
+    @Operation(description = "스태프 코드로 로그인한다.", summary = "스태프 로그인")
+    public ResponseEntity<StaffLoginResponse> login(@RequestBody StaffLoginRequest request) {
+        StaffLoginResponse response = staffAuthService.login(request);
+        return ResponseEntity.ok()
+            .body(response);
+    }
+
+    @PostMapping("/member-tickets/validation")
+    @Operation(description = "스태프가 티켓을 검사한다.", summary = "티켓 검사")
+    public ResponseEntity<TicketValidationResponse> validate(
+        @RequestBody @Valid TicketValidationRequest request,
+        @Staff Long festivalId) {
```

Comment:
```suggestion
        @Staff Long staffId) {
```
또한 entryService의 validate 메서드 또한, 파라미터 명이 잘못된 것 같네요!

Reply:
코드로 인증할 때, jwt 토큰에 StaffCode 정보를 담지 않고 Festival 정보를 담아서 festivalId를 의도한게 맞긴합니다..!!

Reply:
Staff정보를 담는게 더 좋을까요 ?!
그렇게 하면 Staff -> Festival 한 단계 타고 가야해서 이렇게 설계했었어요!

Reply:
지금보니 Staff 도메인의 Staff 엔티티가 `StaffCode`로 되어 있군요 😂
`Staff`로 이름을 바꾸는게 더 명확해보이네요.
지금 같은 방법을 사용하면 티켓을 검증할 때 바로 festivalId로 `MemberTicket` 엔티티에서 `belongsFestival` 메서드를 호출하여 추가적인 쿼리문 없이 티켓을 검증할 수 있겠네요!
하지만 코드 상에서 흐름을 읽기가 더 어려워졌지 않나 싶습니다.. (컨트롤러에서 `@Staff`로 받아온 Long 타입은 staff의 Id라고 당연하게 생각할 수 있으므로)

Reply:
7e8a268 staffId를 넘기는 방식으로 반영완료했습니다!